src/main/java/sim/model/CoffeeShop.java

```diff
@@ -24,11 +25,12 @@
     public CoffeeShop(int numStaff) {
         // Producer inserts input file of orders into shared queue for staff
         Thread producer = new Thread(new Producer(new File("data/orders.csv"), customers));
+        Thread priorityProducer = new Thread(new Producer(new File("data/orders.csv"), customers));
```

Comment:
You just need to pass the priorityOrders queue to the producer instead of the regular customers queue here and I think the rest looks good 👍

Reply:
Oops. It wasn't committed. My bad.

Reply:
![image](https://user-images.githubusercontent.com/65177718/112760364-9cc3d400-8fee-11eb-91d7-9541a44d868b.png)

Oops, it wasn't committed, my bad